suite-common/connect-init/src/slice.ts

```diff
@@ -37,13 +33,7 @@ export const prepareConnectInitThunk = ({
     };
     initSettings: { manifest: Manifest } & Partial<ConnectSettings>;
 }) =>
-    createAsyncThunk<
-        void,
-        void,
-        {
-            state: any;
-        }
-    >(CONNECT_INIT_ACTION_TYPE, async (_, { dispatch, getState }) => {
+    createThunk(CONNECT_INIT_ACTION_TYPE, async (_, { dispatch, getState }) => {
```

Comment:
Btw. does it make sense to have `prepareConnectInitThunk` still here when we have extra dependencies concept with redux-utils implemented now? `actions` and `selectors` can be moved to extra under these keys sections and initSettings moved to `utils` (platform-specific things) maybe?

Reply:
It is not, but want to keep this simple. There will be follow up